src/languages/es_ES.json

```diff
@@ -0,0 +1,232 @@
+    "VLD_002": "La acción es inesperada según el borrador",
+    "VLD_100": "La civilización ya ha sido bloqueada globalmente",
+    "VLD_101": "La civilización ya ha sido bloqueada para este jugador",
+    "VLD_102": "La civilización ya ha sido elegida exclusivamente para este jugador",
+    "VLD_103": "La civilización ya ha sido elegida globalmente por algún jugador",
+    "VLD_200": "La civilización ya ha sido bloqueada exclusivamente por el mismo jugador",
+    "VLD_300": "La civilización no ha sido elegida antes por el oponente",
+    "VLD_301": "The opponent does not have a non-sniped pick of the civilisation",
+    "VLD_900": "El servidor no pudo procesar el borrador correctamente",
+    "VLD_901": "El Borrador contiene una cantidad inválida de turnos paralelos",
+    "VLD_902": "El Borrador contiene un turno paralelo que no requiere la interacción del jugador",
+    "VLD_903": "El Borrador contiene un turno paralelo que no prosigue de un turno que requiere la interacción del jugador",
+    "VLD_904": "El último turno del borrador es un turno paralelo, lo cual es imposible",
+    "VLD_905": "El Borrador posee un turno paralelo que lo prosigue un turno del mismo jugador",
+    "VLD_906": "El Borrador contiene un bloqueo oculto y una selección sin una revelación entre medio",
+    "VLD_907": "El Borrador contiene un turno oculto que nunca es revelado",
+    "VLD_908": "El Borrador no contiene turnos"
+  },
+  "menu": {
+    "welcome": "Bienvenido",
+    "hostOrJoin": "Crea o Únete",
+    "spectate": "Espectar",
+    "practice": "Practicar",
+    "howItWorks": "Como funcióna"
+  },
+  "instructions": {
+    "home": {
+      "title": "Instrucciónes",
+      "1": "Captains mode es un selector de civilizaciónes basado en turnos para emparejamientos. Cada capitán puede <1>elegir</1> y <3>bloquear</3> civilizaciónes en un orden predefinido. Los bloqueos previenen a los capitánes oponentes de elegir una determinada civilización.",
+      "2": "Cada capitán tiene <1>30 segundos</1>. En caso de pasarse el tiempo, una civilización es elegida o baneada al azar.",
+      "3": "También hay extensiónes para los torneos:",
+      "4": "<0>Ocúlto</0> permite ocultar las selecciónes de civilizaciónes hasta que son mostradas explicitamente (visible en el cronograma en la parte superiór).",
+      "5": "<0>Global</0> elegir o bloquear civilizaciónes deshabilita la misma para los turnos siguientes. Para otras reglas específicas, leer la descripción del borrador.",
+      "6": "<0>Práctica</0> es para pruebas, donde ambos capitánes son controlados por el mismo usuario."
+    },
+    "basics": {
+      "title": "Resúmen",
+      "1": "$t(instructions.home.1)",
+      "2": "$t(instructions.home.2)"
+    },
+    "hostAndGuestTurns": {
+      "title": "Turnos de anfitrión y invitado",
+      "1": "Los turnos tienen una acción específica (<1>ELEGIR</1>, <3>BLOQUEAR</3>, <5>VETAR</5>) y algunos posibles modificadores.",
```

Comment:
@HSZemi Should we translate `PICK`, `BAN` and `SNIPE` in this sentence or leave them as it is.

Reply:
We should either translate them nowhere or everywhere. Currently, the english words PICK, BAN, and SNIPE are being used everywhere.

Reply:
My suggestion is to keep it the same throughout the instructions and across languages.

Reply:
Ok, also i think if we mantain this words in the Help section, also on the Draft it should have the same verbs.
`Esperando a el otro capitán a elegir…`  should be `Esperando a el otro capitán a pickear…`  .

Its important to clarify that we use the word "pickear" as the action of pick,"banear", and "snipear" as the actions to ban and snipe, so i suggest using them when needed, or we can use simply "pick/ban/snipe" in that sentences to remain more consistent on using the same words. 

What do you think about this ?

Reply:
> Ok, also i think if we mantain this words in the Help section, also on the Draft it should have the same verbs.
> `Esperando a el otro capitán a elegir…` should be `Esperando a el otro capitán a pickear…` .

Agree.

> Its important to clarify that we use the word "pickear" as the action of pick,"banear", and "snipear" as the actions to ban and snipe, so i suggest using them when needed, or we can use simply "pick/ban/snipe" in that sentences to remain more consistent on using the same words.

I support your suggestion to clarify the meaning of the words pick/ban/snipe in Spanish. I feel that it is fine to use translation of the verbs in the sentences but not in the actual colored tags. The colored tags should always say `PICK`, `BAN` and `SNIPE` for the sake of consistency.